src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj

```diff
@@ -7,7 +7,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Fabric.Admin</PackageId>
         <Description>Provides developers with libraries for the fabric admin under AzureStack to manage fabric infrastructure.</Description>
-        <Version>0.3.0-preview</Version>
+        <Version>0.4.0-preview</Version>
         <AssemblyName>Microsoft.AzureStack.Management.Fabric.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;Fabric;REST HTTP client;azureostackfficial;netcore452</PackageTags>
         <PackageReleaseNotes>
```

Comment:
Please update `PackageReleaseNotes`
Also, please set   `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` and fix all errors reported

Reply:
Fixed.